what4/src/What4/Utils/Process.hs

```diff
@@ -75,6 +75,6 @@ withProcessHandles path args mcwd action = do
               (\(_ :: SomeException) -> return ())
   let onError (_,_,_,ph) = do
         -- Interrupt process; suppress any exceptions that occur.
-        catch (interruptProcessGroupOf ph) (\(_ :: SomeException) -> return ())
+        catch (terminateProcess ph) (\(_ :: SomeException) -> return ())
```

Comment:
What's the rationale for this change? The documentation suggests that it might be an improvement, but potentially not on Windows. Have you done any experiments there?

Reply:
This is catching a missed case from #400, which made this change more widely.  Briefly, `interruptProcessGroupOf` is basically Ctrl+C, while `terminateProcess` is `kill -15` (soft terminate, not `kill -9`). I made the change because some solvers have special (and incorrect) handling of Ctrl+C that makes it not terminate the process.  The one that bit me was that yices tries to be fancy with Ctrl+C handling, but actually doesn't support Ctrl+C interrupts in MC-SAT mode.